buildSrc/src/main/kotlin/net/skyscanner/backpack/tokens/BpkColors.kt

```diff
@@ -138,23 +139,26 @@ private val ColorClass = ClassName("androidx.compose.ui.graphics", "Color")
 
 private const val isLightProperty = "isLight"
 
-private fun String.toComposeStaticName() =
-  CaseFormat.UPPER_UNDERSCORE.to(CaseFormat.UPPER_CAMEL, removePrefix("MARCOMMS_"))
+private fun String.toHexColorBlock() =
+  buildCodeBlock { add("%T(%L)", ColorClass, toHexColor()) }
 
 @OptIn(ExperimentalStdlibApi::class)
+private fun String.toHexColor() =
+  "0xFF${uppercase()}"
+
 private fun toStaticCompose(
   source: BpkColors,
   namespace: String,
 ): TypeSpec {
 
-  fun BpkColorModel.toProperty() : PropertySpec {
+  fun BpkColorModel.toProperty(): PropertySpec {
 
-    fun String.toHexColor() =
-      "0xFF${uppercase()}"
+    fun String.toComposeStaticName() =
+      CaseFormat.UPPER_UNDERSCORE.to(CaseFormat.UPPER_CAMEL, removePrefix("MARCOMMS_"))
```

Comment:
Does this mean we are making the Marcomms colours available in Android? As these should not be available in code as its for marketing material only